### ISSUE TITLE
Duplicate code

### ISSUE THREAD

**Issue by tanyaionova** [jarun/nnn#295]
Seems like selecting opener is duplicated.
Line numbers from `ge93abcf`:
`nnn.c:4555-4557` is exact copy of `nnn.c:4640-4642`:

```c
/* Get custom opener, if set */
opener = xgetenv(env_cfg[NNN_OPENER], utils[OPENER]);
DPRINTF_S(opener);
```
I am not sure, but I believe the second copy is redundant as they both are in the same scope and opener is not changed between them.

**Comment by jarun** [comment]
Closed by PR #296.

**Comment by jarun** [comment]
Thanks for catching this! :+1:
